graphs/templates/graphs/notifications.html

```diff
@@ -0,0 +1,39 @@
+{% extends 'base.html' %}
+{% block content %}
+{% load staticfiles %}
+
+<div class="panel panel-primary">
+  <div class="panel-heading">
+    <h3 class="panel-title">Group1</h3>
+  </div>
+  <table class="table">
+    <tr> 
+    <td> <a href="">seth.mridul@gmail.com shared graph some_name_of_graph with the group Group 1.</td>
```

Comment:
Shouldn't there be generic variables here instead of seth.mridul@gmail.com, the name of the graph, and the name of the group?